local.Dockerfile

```diff
@@ -9,8 +9,9 @@ RUN npm install
 
 COPY . ./
 
-RUN npm build
+RUN npm run build
 
+RUN apk --update --no-cache add curl
```

Comment:
What are we using `curl` for within the `local.Dockerfile` ?

Reply:
curl is used for the health check for the cucumber test container to ensure the web is up and running to then continue to run the tests as there was issues with the web not being up in time for the cucumber to run